pkg/loop/internal/relayer.go

```diff
@@ -201,6 +203,9 @@ func (r *relayerClient) NewPluginProvider(ctx context.Context, rargs types.Relay
 			},
 		})
 		if err != nil {
+			if rargs.ProviderType == string(types.Median) {
+				return 0, nil, types.UnwrapClientError(err)
+			}
```

Comment:
Why limit this to median? Seems brittle and easy to forget.
```suggestion
			return 0, nil, types.UnwrapClientError(err)
```

Reply:
After removing the conversion between `errors.ErrUnsupported` and `codes.Unimplemented`, I decided it's best to just remove this.  Absent that, all the `UnwrapClientError` method does is convert the type of the error from a generic gRPC error type to type `ChainReaderError`.  We could keep it in there and rename it to something like `PluginProviderError`, but since nothing checks the type on the client side anyway, it wouldn't matter much. Presumably, the errors defined in `ChainReaderError` will eventually be things a product plugin will look for to see if there were specific problems with ChainReader, but presently I think these errors are currently only being checked for and handled in some of @nolag 's other PR's.

Retrospectively I shouldn't have called it here, because there could be errors unrelated to ChainReader popping up, even with it restricted to `MedianProvider`.